AWSIoT/Internal/AWSIoTMQTTClient.h

```diff
@@ -91,6 +91,7 @@
 @property(atomic, assign) BOOL isMetricsEnabled;
 @property(atomic, assign) NSUInteger publishRetryThrottle;
 @property(atomic, strong) NSString *userMetaData;
+@property(atomic, copy) NSString *password;
```

Comment:
Why `copy` rather than `strong`, and would it then make sense to update `userMetaData` to `copy` as well?

Reply:
There was a lot of discussion years ago that it's generally a good rule of thumb to use copy for NSString.
https://stackoverflow.com/questions/387959/nsstring-property-copy-or-retain

I was trying to reduce the blast radius of my change, but really, once a user sets a value, they should not be able to modify it.  I will update userMetaData to use `copy`